### ISSUE TITLE
Feature Request : metis output

### ISSUE THREAD

**Issue by h4ck3rm1k3** [eXascaleInfolab/PyNetConvert#5]
I would like a metis output function.

**Comment by luav** [comment]
Yes, it would be nice to have conversion also to the Metis format but it is out of my priorities. This app has been develop to convert DIMACS10 (Metis format) and some other networks to the [NSL/NCOL](http://www.lumais.com/docs/format.nsl) or [RCG](http://www.lumais.com/docs/format.rcg) for the subsequent clustering and [benchmarking of the clustering algorithms](https://github.com/eXascaleInfolab/PyCABeM).

You are welcome to contribute and add this feature.

**Comment by h4ck3rm1k3** [comment]
I am attempting to use https://metis.readthedocs.io/en/latest/ and will see how that works first.
